src/Utility.ts

```diff
@@ -169,9 +168,19 @@ export namespace Utility {
         const builder: xml2js.Builder = new xml2js.Builder();
         const newXml: string = builder.buildObject(jsonObj);
         await fse.writeFile(serverXml, newXml);
-    }/* tslint:enable:no-any */
+    }
+
+    export async function copyServerConfig(source: string, target: string): Promise<void> {
+        const xml: string = await fse.readFile(source, 'utf8');
+        const jsonObj: {} = await parseXml(xml);
+        const builder: xml2js.Builder = new xml2js.Builder();
+        const newXml: string = builder.buildObject(jsonObj);
+        if (!await fse.pathExists(target)) {
```

Comment:
Can we use fse.ensureFile (https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureFile.md)